src/components/add-form/index.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
-import RadioInput from '../radio-input';
 import './styles.css';
 
+const TimeFrames = {
+  soon: '7',
+  kindOfSoon: '14',
+  notSoon: '30',
+};
+
```

Comment:
Lol! i missed this when I created the new time frames, thaaaanks!!!

Reply:
This object is so great and useful! 💯